api/src/main/resources/messages.properties

```diff
@@ -17,4 +17,6 @@ ${project.parent.artifactId}.settings=Settings
 ${project.parent.artifactId}.saved=Global Properties Saved
 ${project.parent.artifactId}.appBaseUrl.label = App Base URL
 ${project.parent.artifactId}.appFolderPath.label = App Folder Path
-${project.parent.artifactId}.appStoreUrl.label = App Store URL
\ No newline at end of file
+${project.parent.artifactId}.appStoreUrl.label = App Store URL
+${project.parent.artifactId}.invalid_owa_Url = The OWA installation link passed is invalid
+${project.parent.artifactId}.missing_owa_file = Installation link doesnot specify download file
```

Comment:
What is the meaning of link to the end user?

Reply:
Changed it to URL